nastavnik/dodavanje_asistenata.php

```diff
@@ -0,0 +1,153 @@
+
+                $angazman_status = 3;
+                $nivo_prist = 'asistent';
+            }else if($uloga == 2){ // Asistent
+                // U tabelu angazman dodajemo angazman_status = 2 => Asistent
+                // U tabelu nastavnik_predmet dodajemo nivo_pristupa = asistent => Asistent
+
+                $angazman_status = 2;
+                $nivo_prist = 'asistent';
+            }else if($uloga == 3){ // Super asistent
+                // U tabelu angazman dodajemo angazman_status = 2 => Asistent
+                // U tabelu nastavnik_predmet dodajemo nivo_pristupa = super_asistent => Super asistent
+
+                $angazman_status = 2;
+                $nivo_prist = 'asistent';
+            }
+
+            // Unesimo ili uredimo angažman
+            if(!$angazman){
+                db_query("INSERT into angazman set akademska_godina = $akademska_godina, osoba = $osoba, angazman_status = $angazman_status, predmet = ".$_GET['predmet']);
+            }else{
+                db_query("UPDATE angazman set angazman_status = $angazman_status where akademska_godina = $akademska_godina and osoba = $osoba and predmet = ".$_GET['predmet']);
+            }
+
+            // Unesimo ili uredimo nivo pristupa
+            if(!$nivo_pristupa){
+                db_query("INSERT INTO nastavnik_predmet SET nastavnik = $osoba, akademska_godina = $akademska_godina, nivo_pristupa = '$nivo_prist', predmet = ".$_GET['predmet']);
+            }else{
+                db_query("UPDATE nastavnik_predmet SET nivo_pristupa = '$nivo_prist' where nastavnik = $osoba");
+                //db_query("UPDATE nastavnik_predmet SET nivo_pristupa = 12 where nastavnik = $osoba and akademska_godina = $akademska_godina and predmet = ".$_GET['predmet']);
+            }
+        }
+    }
+
+    $angazovane_osobe = db_query("select o.ime, o.prezime, ang.naziv from angazman as an inner join angazman_status as ang on an.angazman_status = ang.id inner join osoba as o on an.osoba = o.id where an.akademska_godina = $akademska_godina and an.predmet = ".$_GET['predmet']);
+    //$angazovane_osobe = db_query("select * from angazman as an where an.akademska_godina = $akademska_godina and an.predmet = ".$_GET['predmet']);
+
+
+    ?>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.10/js/select2.min.js"></script>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.10/css/select2.min.css" rel="stylesheet"/>
+
+    <br><br>
+    <p><h3>Dodajte asistenta / demonstratora na predmet</h3></p>
+    <p>
+    <form method="post">
+        <select name="osoba" class="users-search">
+            <option value="0">Odaberite osobu</option>
+            <?php
+            while ($o = db_fetch_row($osobe)) {
+                ?>
+                <option value="<?php echo $o[0]; ?>" <?php if(isset($osoba)){if($osoba == $o[0]) echo 'selected';} ?> >
```

Comment:
Zašto ne if (isset($osoba) && $osoba == $o[0]) ?? U PHPu je ista kratkospojna evaluacija kao u C i C++ :)